legal-api/migrations/versions/c4732cd8abfd_jurisdiction.py

```diff
@@ -0,0 +1,97 @@
+"""jurisdiction
+
+Revision ID: c4732cd8abfd
+Revises: ef8f033d317b
+Create Date: 2024-05-24 14:38:45.440604
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'c4732cd8abfd'
+down_revision = 'ef8f033d317b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('jurisdictions',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('country', sa.String(length=10), nullable=False),
+                    sa.Column('region', sa.String(length=10), nullable=True),
+                    sa.Column('identifier', sa.String(length=50), nullable=True),
+                    sa.Column('legal_name', sa.String(length=1000), nullable=True),
+                    sa.Column('tax_id', sa.String(length=15), nullable=True),
+                    sa.Column('incorporation_date', sa.DateTime(timezone=True), nullable=True),
+                    sa.Column('expro_identifier', sa.String(length=10), nullable=True),
+                    sa.Column('expro_legal_name', sa.String(length=1000), nullable=True),
+                    sa.Column('business_id', sa.Integer(), nullable=False),
+                    sa.Column('filing_id', sa.Integer(), nullable=False),
+                    sa.ForeignKeyConstraint(['business_id'], ['businesses.id'], ),
+                    sa.ForeignKeyConstraint(['filing_id'], ['filings.id'], ),
+                    sa.PrimaryKeyConstraint('id')
+                    )
```

Comment:
After our conversation about jurisdictions and how it's a lot more extensively modelled to support various uses cases in COLIN, I think I agree that we need to do more work here.  

From looking at some of the requirements coming out of some of the involuntary dissolution eligibility criteria, XPRO, Continuation In/Out, looking at COLIN data model and etc, I think we might want to look into this more.

It might make sense to model things similar to how COLIN models jurisdictions.  Or at least be able to cover the same scenarios.  Seems to be a lot of scenarios.

Jurisdiction related COLIN tables

![286254E0](https://github.com/bcgov/lear/assets/6685223/d67eab95-64ad-4578-824f-9fe2544506ef)

![1D687095](https://github.com/bcgov/lear/assets/6685223/ac7317b2-fa92-4318-b444-920cb8e132b3)

![3991B3BA](https://github.com/bcgov/lear/assets/6685223/5fc16ec8-4ba2-4b5d-8358-227c764d4f98)

Reply:
as per our conversation, we will leave as is for now.  May revisit or extend jurisdiction table later.